### PR TITLE
Generic getting started

### PR DESCRIPTION
**Pull request by yaelpeled**

### CODE REVIEW

File: source/guide/3.2/quickstart-real-cloud.md [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+# Overview
+
+In this tutorial you will bootstrap a Cloudify manager on a cloud environment and install a sample Cloudify blueprint on it.
+
+This tutorial shows how to bootstrap a Cloudify manager on:
+ 
+  - [OpenStack](plugin-openstack.html)
+  - [Softlayer](softlayer-openstack.html)
+
+The blueprint you'll be deploying, describes a nodejs application that connects to a MongoDB database and presents a wine catalog.
```
Comment (ran-z) [inline source/guide/3.2/quickstart-real-cloud.md]: no need for comma

File: source/guide/3.2/quickstart-real-cloud.md [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+
+  - [openstack nodecellar blueprint]({{page.openstack_blueprint_file_link}})
+  - [softlayer nodecellar blueprint]({{page.softlayer_blueprint_file_link}})
+
+To learn more about blueprint syntax and elements please refer to the [Blueprint Authoring Guide](guide-blueprint.html).
+
+# Before You Begin
+
+It is recommended that you try the [Getting started](quickstart.html) first familiarize
```
Comment (ran-z) [inline source/guide/3.2/quickstart-real-cloud.md]: ".. that you try the 'Getting started guide' to first ..."

File: source/guide/3.2/quickstart-openstack.md [hunk 3]
```diff
@@ -1,31 +1,54 @@
 ---
```
Comment (ran-z) [inline source/guide/3.2/quickstart-openstack.md]: is this some github issue? according to my understanding this file shouldnt exist anymore correct?
quickstart-openstack was renamed quickstart-real-cloud?

File: source/guide/3.2/quickstart-real-cloud.md [hunk 4]
```diff
@@ -1,31 +1,41 @@
...
-To learn more about blueprint syntax and elements please refer to the [Blueprints Guide]({{blueprint_guide_link}}).
+  - [openstack nodecellar blueprint]({{page.openstack_blueprint_file_link}})
+  - [softlayer nodecellar blueprint]({{page.softlayer_blueprint_file_link}})
+
+To learn more about blueprint syntax and elements please refer to the [Blueprint Authoring Guide](guide-blueprint.html).
 
 # Before You Begin
 
-It is recommended that you try the [Getting started](quickstart.html) first familiarize
+It is recommended that you try the [Getting started guide](quickstart.html) first familiarize
```
Comment (nir0s) [inline source/guide/3.2/quickstart-real-cloud.md]: to familiarize

File: source/guide/3.2/quickstart-real-cloud.md [hunk 5]
```diff
@@ -35,9 +45,9 @@ The first thing you'll need to do is install the Cloudify CLI,
 which will let you upload blueprints, create deployments, and execute workflows.
 To do so follow the steps described in the [CLI installation guide](installation-cli.html).
 
-## Step 2: Download the Manager Blueprint for OpenStack
+## Step 2: Download the Manager Blueprint
 
-Next, let's create a cloudify-manager dir and download the OpenStack Manager Blueprint.
+Next, let's create a cloudify-manager dir and download the Manager Blueprint.
```
Comment (nir0s) [inline source/guide/3.2/quickstart-real-cloud.md]: directory? :)

File: source/guide/3.2/quickstart-real-cloud.md [hunk 6]
```diff
@@ -1,31 +1,41 @@
...
 # Overview
 
-In this tutorial you will bootstrap a Cloudify manager in your OpenStack environment
-and install a sample Cloudify blueprint on it.
+In this tutorial you will bootstrap a Cloudify manager on a cloud environment and install a sample Cloudify blueprint on it.
+
+This tutorial shows how to bootstrap a Cloudify manager on:
+ 
+  - [OpenStack](plugin-openstack.html)
+  - [Softlayer](softlayer-openstack.html)
```
Comment (nir0s) [inline source/guide/3.2/quickstart-real-cloud.md]: I think you meant plugin-softlayer.html ?

File: source/guide/3.2/quickstart-real-cloud.md [hunk 7]
```diff
@@ -1,31 +1,41 @@
...
+
+The blueprint you'll be deploying describes a nodejs application that connects to a MongoDB database and presents a wine catalog.
 
-The [blueprint]({{page.blueprint_file_link}}) you'll be deploying,
-describes a nodejs application that connects to a MongoDB database and presents a wine catalog.
-To learn more about blueprint syntax and elements please refer to the [Blueprints Guide]({{blueprint_guide_link}}).
+  - [openstack nodecellar blueprint]({{page.openstack_blueprint_file_link}})
```
Comment (nir0s) [inline source/guide/3.2/quickstart-real-cloud.md]: OpenStack...

File: source/guide/3.2/quickstart-real-cloud.md [hunk 8]
```diff
@@ -1,31 +1,41 @@
...
+  - [softlayer nodecellar blueprint]({{page.softlayer_blueprint_file_link}})
```
Comment (nir0s) [inline source/guide/3.2/quickstart-real-cloud.md]: SoftLayer

File: source/guide/3.2/quickstart-real-cloud.md [hunk 9]
```diff
@@ -1,31 +1,41 @@
...
+
+To learn more about blueprint syntax and elements please refer to the [Blueprint Authoring Guide](guide-blueprint.html).
```
Comment (nir0s) [inline source/guide/3.2/quickstart-real-cloud.md]: And possible to the DSL documentation...

**Comment by nir0s** [comment]
This is brilliant.

I really think we should use this rather then have a page for every Cloud. Let's discuss ASAP.

**Comment by nir0s** [comment]
Also, please solve the conflicts :)

**Pull request merged**